src/java.base/share/classes/sun/security/ec/ECPrivateKeyImpl.java

```diff
@@ -25,7 +25,7 @@
 
 /*
  * ===========================================================================
- * (c) Copyright IBM Corp. 2022, 2023 All Rights Reserved
+ * (c) Copyright IBM Corp. 2022, 2024 All Rights Reserved
```

Comment:
It appears we no longer need any change to this file nor `ECPublicKeyImpl.java` relative to upsteam. Please fix that here and create a follow-up for https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/792 that does the same in the head stream.

Reply:
Removed them. The copyrights check is failing, but I guess that's expected.

Reply:
Created a PR for `OpenJDKnext` to update it there too: https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/795